src/main/capi/data_chunk-c.cpp

```diff
@@ -46,6 +46,32 @@ void duckdb_data_chunk_reset(duckdb_data_chunk chunk) {
 	dchunk->Reset();
 }
 
+void duckdb_data_chunk_copy(duckdb_data_chunk src, duckdb_data_chunk dst) {
+	auto dsrc = reinterpret_cast<duckdb::DataChunk *>(src);
+	auto ddst = reinterpret_cast<duckdb::DataChunk *>(dst);
+	dsrc->Copy(*ddst);
+}
```

Comment:
Sorry to reiterate on this again, but after some internal discussion, we'd prefer to remove this as well - as it is a thin wrapper for what we can already do with `duckdb_data_chunk_get_column_count`, `duckdb_data_chunk_get_vector`, and `duckdb_vector_copy_sel` (by passing an incremental selection vector).